contracts/Core/AssetManager.sol

```diff
@@ -125,12 +124,13 @@ contract AssetManager is ACL, AssetStorage, Constants {
         uint32 aggregationMethod,
         int8 power,
         string calldata name
-    ) external onlyRole(ASSET_MODIFIER_ROLE) {
+    ) external onlyRole(ASSET_MODIFIER_ROLE) checkStates(State.Dispute, State.Confirm, parameters.epochLength()) {
         require(aggregationMethod > 0 && aggregationMethod < parameters.aggregationRange(), "Aggregation range out of bounds");
 
         require(jobIDs.length > 1, "Number of jobIDs low to create collection");
 
         numAssets = numAssets + 1;
+        numActiveAssets++;
```

Comment:
We should keep the increment uniform, in line 132 and 133 for better readability.